freqtrade/constants.py

```diff
@@ -243,7 +243,8 @@
                     }
                 }
             },
-            'required': ['enabled', 'token', 'chat_id']
+            'required': ['enabled', 'token', 'chat_id'],
+            'balance_dust_level': 0.0001
```

Comment:
that's ... not quite how you add a parameter to the validation dict...

```suggestion
            'balance_dust_level': {'type': 'number', minimum: 0.0},
```

The code as you have it should go to the full config example (and maybe to the other examples, but i'll leave that up to you).